docs/source/tutorials/cmip6-ecs.ipynb

```diff
@@ -0,0 +1,3690 @@
+{
```

Comment:
<p>How about we rename <code>col_subset</code>  to <code>cat_subset</code>  so as to avoid creating confusion when referring to the loaded collection in the next cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds-funnel/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>+1</p>